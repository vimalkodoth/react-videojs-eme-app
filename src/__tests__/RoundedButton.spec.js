import React from "react";

describe("RoundedButton", () => {
    it("", () => {});
});
