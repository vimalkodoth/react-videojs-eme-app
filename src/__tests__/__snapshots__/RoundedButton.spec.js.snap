// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoundedButton renders correctly 1`] = `
<EmotionCssPropInternal
  __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
  className="rounded"
  css={
    Object {
      "name": "1in5v9c-RoundedButtonStyles",
      "styles": "&.rounded{position:absolute;width:100%;top:30%;display:flex;justify-content:center;& .action{color:#fff;text-align:center;& .text{display:block;}}& .element{display:flex;width:100px;background:#fff;height:100px;border-radius:50%;margin-bottom:5px;justify-content:center;align-items:center;cursor:pointer;& .right-arrow{width:0;height:0;border-top:25px solid transparent;border-bottom:25px solid transparent;border-left:40px solid #000;display:block;margin-left:15px;transition:border-left-color 0.45s ease-in;&:hover,&:focus{border-left-color:red;}}& .title{margin:10px 0;}}};label:RoundedButtonStyles;",
    }
  }
  data-testid="rounded-button"
>
  <span
    className="action"
  >
    <Link
      to="/link"
    >
      <span
        className="element"
      >
        <span
          className="right-arrow"
        />
      </span>
    </Link>
    <span
      className="text"
    >
      Hello world!
    </span>
  </span>
</EmotionCssPropInternal>
`;
